res/WELCOME.md

```diff
@@ -15,7 +15,17 @@ It is in general a good idea to await the automated test ("Checks") in this Pull
 
 ## Contributing the patches
 
-Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`.
+Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`. A good way to find other contributors is to locate recent pull requests where someone has been `/allow`ed:
```

Comment:
The default for `/allow` is to allow the PR author.  Should the `<username>` parameter be removed?

Just mentioning it while this is being updated.  Understand it is not really part of this change.

Reply:
It would take me about 2 minutes to fold that into the commit, @dscho @nasamuffin opinions?

Reply:
I'd love to have this change, but in a separate commit (although still in this PR).

Reply:
Done. :-)